calyx-py/calyx/builder.py

```diff
@@ -1,50 +1,3 @@
-from __future__ import annotations
```

Comment:
Not sure if this was intentional

Reply:
You're right. I was actually using this file as a way to keep track of which builder features I had created sections for (to ensure full coverage). I committed this in error, sorry! will amend that!